### DISCUSSION TITLE
After second login frontend user info only gets updated in sidebar, not in index

### DISCUSSION THREAD

**Discussion by liutec** [fastapi/full-stack-fastapi-template#1200] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
# components/common/Sidebar.tsx
[..]
const Sidebar: React.FC = () => {
  const queryClient = useQueryClient()
  const bgColor = useColorModeValue('ui.white', 'ui.dark')
  const textColor = useColorModeValue('ui.dark', 'ui.white')
  const secBgColor = useColorModeValue('ui.secondary', 'ui.darkSlate')
  const currentUser = queryClient.getQueryData<UserOut>('currentUser')
  const { isOpen, onOpen, onClose } = useDisclosure()
  const { logout } = useAuth()

  console.log(currentUser)  // outputs info for the currently authenticated user
[...]

# routes/_layout/index.tsx
[...]
function Dashboard() {
  const queryClient = useQueryClient()

  const currentUser = queryClient.getQueryData<UserOut>('currentUser')

  console.log(currentUser)  // outputs info for the previously authenticated user
[...]
```

### Description

Context:
- create one or more additional users

Steps to reproduce:
- login as user 1
- log out
- login as user 2
- the sidebar correctly displays the newly authenticated user (user 2)
- the dashboard welcomes the previously authenticated user (user 1)

### Operating System

Linux

### Operating System Details

not OS related

### Python Version

3.10

### Additional Context

![image](https://github.com/tiangolo/full-stack-fastapi-template/assets/3606208/229473af-bb0c-4d66-9e77-783e8b433ca6)

This issue reproduces everywhere useAuth() is *not* called and queryClient.getQueryData<UserOut>('currentUser') is used.

**Comment by YuriiMotov** [comment]
It was fixed somethere. Works well in current version

<img width="942" height="276" alt="image" src="https://github.com/user-attachments/assets/8c0e457b-4c0c-4697-b26a-ede732c562ba" />


Thanks for reporting!
